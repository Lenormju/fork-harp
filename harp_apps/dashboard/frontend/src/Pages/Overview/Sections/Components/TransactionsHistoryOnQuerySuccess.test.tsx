import { render } from "@testing-library/react"
import { expect, it } from "vitest"

import {
  TransactionsHistoryOnQuerySuccess,
  TransactionsHistoryOnQuerySuccessProps,
} from "./TransactionsHistoryOnQuerySuccess.tsx"

it("renders without crashing", () => {
  const data: TransactionsHistoryOnQuerySuccessProps["data"] = {
    meanDuration: 2000,
    meanApdex: 85,
    errors: { count: 2, rate: 0.05 },
    transactions: [],
    timeRange: "month",
    count: 100,
  }

  const { container } = render(
    <TransactionsHistoryOnQuerySuccess data={data} title="Test Title" className="test-class" />,
  )
  expect(container).toMatchSnapshot()
})
