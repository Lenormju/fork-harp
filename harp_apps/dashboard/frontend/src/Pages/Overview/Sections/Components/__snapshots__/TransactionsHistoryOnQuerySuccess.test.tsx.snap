// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders without crashing 1`] = `<div />`;
